report-ui/style.css

```
@@ -334,14 +342,25 @@ body {
+  /* Top-aligned, not centred: chips are stretched to a common height, so
+     centring would drop the number and label of a chip without a sub-line
+     out of line with its neighbours. */
+  justify-content: flex-start;
   gap: 4px;
   min-width: 92px;
-  padding: 8px 8px 6px;
+  /* The sub-line strip cancels these to bleed to the chip edges, so they are
+     tokens rather than literals repeated across the breakpoints. */
+  --chip-pad-x: 8px;
+  --chip-pad-b: 6px;
+  padding: 8px var(--chip-pad-x) var(--chip-pad-b);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)